lms/views/admin/application_instance/analytics.py

```diff
@@ -0,0 +1,16 @@
+from pyramid.view import view_config
+
+from lms.security import Permissions
+from lms.views.admin.application_instance._core import BaseApplicationInstanceView
+
+
+class AnalyticsApplicationInstanceView(BaseApplicationInstanceView):
+    @view_config(
+        route_name="admin.instance.analytics",
+        renderer="lms:templates/admin/application_instance/analytics.html.jinja2",
+        permission=Permissions.STAFF,
```

Comment:
It's possible this route needs to eventually be accessible by other roles, but for now, since the entry point will be in the admin, I have decided to add this role to avoid leaking data unintentionally.

We can revisit this later on.